templates/hardwarelab/event_manager.html

```diff
@@ -0,0 +1,192 @@
+{% extends "layout.html" %}
+{% block content %}
+<script>
+$(document).ready(function(){
+    $(".create-rental").on("click", function() {
+      $rental_id = $(this).siblings("#rental-id").attr("value");
+      $inventory_id = $(this).parentsUntil('#reservations').find('#inventory_id').val();
+      console.log($inventory_id);
+      if($inventory_id == ''){
+        console.log('empty');
+        alertify.alert("Please select an Inventory ID");
+      } else {
+        $("#create-rental-modal").modal({
+          closable  : true,
+          onDeny    : function(){
+            return true;
+          },
+          onApprove : function() {
+            $.ajax({
+              url : "/api/rent/", // the endpoint
+              type : "GET", // http method
+              data : { 
+                rental_id: $rental_id,
+                inventory_id: $inventory_id
+              },
+              success : function(data) {
+                location.reload();
```

Comment:
That only works on the first replace attempt. Fails the second time.
Fixed by explicitly refreshing only the main container